schemas/v1/29_add_parsed_return_to_receipt_returns.go

```diff
@@ -0,0 +1,15 @@
+package v1
+
+func init() {
+	patches.Register(
+		29,
+		`
+	ALTER TABLE {{ .SchemaName | default "public"}}.receipts
+		ADD COLUMN IF NOT EXISTS "parsed_return" JSONB;
+
+	ALTER TABLE {{ .SchemaName | default "public"}}.receipts
+		ADD COLUMN IF NOT EXISTS "return" bytea;
+	
```

Comment:
Reorder to match the model please, otherwise we might have issues (similar to the actors table..). Thanks!

```suggestion
	ALTER TABLE {{ .SchemaName | default "public"}}.receipts
		ADD COLUMN IF NOT EXISTS "return" bytea;
		
	ALTER TABLE {{ .SchemaName | default "public"}}.receipts
		ADD COLUMN IF NOT EXISTS "parsed_return" JSONB;
	
```

Reply:
Thanks!